docs/ADRs/adr-006-tech-choice-on-quals-right-to-teach.md

```diff
@@ -0,0 +1,30 @@
+## 006 - Choose HL Technical Direction For Quals and Right To Teach Services
+
+**Date: 15-02-2023**
+
+## Status
+
+Proposed
+
+## Context
+
+In order to proceed on iteration 1 of the Teacher Self Serve Portal replacement aka (Qualifications Service / Teacher Account / Teacher Self Serve TBC),
+we want to decide on the following
+1. Proposed tech for each service?
+2. One Front end application (in code repo terms) or two?
+3. How will we abstract away underlying data dependencies while we have high uncertainty on service design?
+4. Datasources?
+
+## Decision
+
+1. This is not a technical issue (i.e. either is fine from a tech POV), this is more a resource decision. TBC but the assumptions at the time of writing are:
+    Right To Teach (Ruby On Rails) + PostgreSQL
+    Qualifications (Ruby On Rails) + PostgreSQL
```

Comment:
Does each service needs it own Postgres DB if the API is doing the heavy lifting?

Reply:
maybe not, its just an assumption at the moment, the tech is marked as TBC